great_expectations/profile/basic_dataset_profiler.py

```diff
@@ -123,7 +123,7 @@ class BasicDatasetProfiler(BasicDatasetProfilerBase):
     such as min, max, mean and median, for numeric columns, and distribution of values, when appropriate.
     """
 
-    @classmethod
+    @classmethod  # noqa: C901
```

Comment:
This is pretty annoying.
flake8 needs `noqa` statements on both the method and `@classmethod` decorator.